vernac/comFixpoint.ml

```diff
@@ -294,7 +294,10 @@ let declare_fixpoint_interactive_generic ?indexes ~scope ?clearbody ~poly ?typin
     Declare.Proof.start_mutual_with_initialization ~info
       evd ~mutual_info:(cofix,indexes,init_terms) ~cinfo:thms None in
   (* Declare notations *)
-  List.iter (Metasyntax.add_notation_interpretation ~local:(scope=Locality.Discharge) (Global.env())) ntns;
+  let _ =
```

Comment:
```suggestion
  let () =
```

Reply:
I improved this part too. Tell me if ok for you. (In particular, overriding the notation together with silencing the overridden warning looks a bit hackish.)

Reply:
@SkySkimmer: I additionally moved the declaration of notations from `comFixpoint.ml` to `declare.ml`, so that the role of each of the file is the same whether the fixpoint is interactive or not.